release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2357,6 +2357,8 @@ rpc error: code = Unknown desc = error pinging docker registry registry.stage.re
 +
 This caused image pulls to fail against the inaccessible private registry, normally intended for internal Red Hat testing, and related Operator installations and upgrades would never succeed. See xref:../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-refresh-subs_olm-deleting-operators-from-a-cluster[Refreshing failing subscriptions] for a workaround to clean up this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909152[*BZ#1909152*])
 
+* Currently, a Kubernetes port collision issue can cause a breakdown in pod-to-pod communication, even after pods are redeployed. To avoid this, don't build your application so that the ports that are causing the issue are hard coded, and don't use the same ports for both cluster and pod IP addresses. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939676[*BZ#1939676*], link:https://bugzilla.redhat.com/show_bug.cgi?id=1939045[*BZ#1939045*])
```

Comment:
The workding of To avoid this, don't build your application so that the ports that are causing the issue are hard coded, is confusing. ... 

Can we frame this better? Can we show an example of what to avoid?

Reply:
Agreed -- will edit for clarity and include link to the [KB article](https://access.redhat.com/solutions/5940711), which gives a detailed. explanation.

Reply:
Changes made as suggested.